plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1706,12 +1706,34 @@ public void testCreateNestedPartitionedTable()
                 " select true, 1, array['uno', 'dos', 'tres'], BIGINT '1', REAL '1.0', DOUBLE '1.0', map(array[1,2,3,4], array['ek','don','teen','char'])," +
                 " CAST(1.0 as DECIMAL(5,2))," +
                 " 'one', VARBINARY 'binary0/1values',\n" +
-                " cast(current_timestamp as TIMESTAMP), (CAST(ROW(null, 'this is a random value') AS ROW(int, varchar))), current_date";
+                " TIMESTAMP '2021-07-24 02:43:57.348000'," +
+                " (CAST(ROW(null, 'this is a random value') AS ROW(int, varchar))), " +
+                " DATE '2021-07-24'";
 
         assertUpdate(insertSql, 1);
         MaterializedResult result = computeActual("SELECT * from test_nested_table_1");
         assertEquals(result.getRowCount(), 1);
 
+        assertThat(query("SHOW STATS FOR test_nested_table_1"))
+                .projected(0, 2, 3, 4, 5, 6) // ignore data size which is available for Parquet, but not for ORC
+                .skippingTypesCheck()
+                .matches("VALUES " +
+                        // TODO (https://github.com/trinodb/trino/issues/8648) the NDV numbers are wrong, should be 1, not 0
```

Comment:
https://github.com/trinodb/trino/issues/8648

Reply:
doesn't projected index 3 map to nulls-fraction which is correctly 0? 

[spec](https://iceberg.apache.org/spec/) mentions that distinct values are deprecated anyway, so keeping them null (index 2) seems fine to me.

Reply:
@phd3 good catch. i fell into my own trap here.

Reply:
I was surprised to see that these were deprecated, but it looks like they were just added back: https://github.com/apache/iceberg/pull/2805

Reply:
@electrum i don't think per-file NDV is _that_ useful, unless query is very selective.

Do we return them today, if they are present?


Since things like https://github.com/apache/iceberg/pull/2805 depend on the writer, i guess we need to have SHOW STATS coverage with Trino and Spark writers.
cc @joshthoward